frontend/ui/data/keyboardlayouts/keypopup/ka_popup.lua

```diff
@@ -0,0 +1,119 @@
+return {
+    _a_ = {
+        "ა",
+        north = "ჺ", -- georgian letter ain
+    },
+    _b_ = {
+        "ბ",
```

Comment:
If there's only one character it's a bit weird to use a popup.

Basically either:
```lua
local _b_ = "ბ"
```

Or:

```lua
            { _B_,    "ბ",    ">",    prd,    "Ó",    "ó",    "†",    "‡", },
```